hack/.ci/run-e2e-gke.sh

```diff
@@ -6,277 +6,33 @@
 set -euExo pipefail
 shopt -s inherit_errexit
 
-if [ -z ${SO_SUITE+x} ]; then
-  echo "SO_SUITE can't be empty" > /dev/stderr
-  exit 2
-fi
+ARTIFACTS="${ARTIFACTS:-$( mktemp -d )}"
```

Comment:
it used to be required, now it silently passes and saves artifacts in temporary location which cannot be easily extracted after CI run. Why this is changed?

Reply:
fixed